hooks/build

```diff
@@ -155,10 +156,22 @@ export DOCKERHUB_REPO=${DOCKERHUB_REPO:="airflow"}
 export PYTHON_VERSION=\
 ${PYTHON_VERSION:=$(python -c 'import sys;print("%s.%s" % (sys.version_info.major, sys.version_info.minor))')}
 
+. ${MY_DIR}/_default_branch.sh
+
+# Source brnnch will be set in DockerHub
+SOURCE_BRANCH=${SOURCE_BRANCH:=${DEFAULT_BRANCH}}
+# if AIRFLOW_CONTAINER_BRANCH_NAME is not set it will be set to either SOURCE_BRANCH (if set in DockerHub
+# or default branch configured in _default_branch.sh
+AIRFLOW_CONTAINER_BRANCH_NAME=${AIRFLOW_CONTAINER_BRANCH_NAME:=${SOURCE_BRANCH}}
```

Comment:
I found similarly looking code in the `scripts/ci//local_ci_stop_environment.sh` file. Do it not need a change? Probably not, but I prefer to ask.

Reply:
Not really. Stable branch is usually only built on Travis CI not for local developer builds. 

We usually work in the way that we make v1-10-test branch work locally  (where DEFAULT_BRANCH=v1-10-test is fine). And then we move stable branch up when v1-10-test branch is green. Then in effect stable branch is only run on Travis.

Reply:
Besides this is "stop" which is not really using the branch.